eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -23,161 +17,64 @@ parameters:
 - name: PublishCodeCoverage
   type: boolean
   default: false
-- name: SubscriptionConfiguration
-  type: string
-  default: $(sub-config-azure-cloud-test-resources)
-- name: ResourceGroupLocation
+- name: PreSteps
+  type: object
+  default: []
+- name: PostSteps
+  type: object
+  default: []
+- name: Location
   type: string
-  default: ""
-- name: ResourceGroupLocationCanary
+  default: ''
+- name: CloudConfig
+  type: object
+  default: {}
+- name: Matrix
   type: string
-  default: "centraluseuap"
-- name: ArmTemplateParameters
+- name: DependsOn
   type: string
-  default: "@{}"
-- name: TestBrowser
-  type: boolean
-  default: true
-- name: TestSamples
-  type: boolean
-  default: true
-- name: TestMinMax
-  type: boolean
-  default: false
-- name: TestSame
+  default: ''
+- name: UsePlatformContainer
   type: boolean
   default: false
-- name: TestCanary
-  type: boolean
-  default: false
-- name: Matrix
-  type: object
-  default:
-    Windows Node 8:
-      Pool: azsdk-pool-mms-win-2019-general
-      OSVmImage: MMS2019
-      TestType: "node"
-      NodeTestVersion: "8.x"
-      PublishCodeCoverage: true
-    Linux Node 10:
-      Pool: azsdk-pool-mms-ubuntu-1804-general
-      OSVmImage: MMSUbuntu18.04
-      TestType: "node"
-      NodeTestVersion: "10.x"
-    Linux Node 12:
-      Pool: azsdk-pool-mms-ubuntu-1804-general
-      OSVmImage: MMSUbuntu18.04
-      TestType: "node"
-      NodeTestVersion: "12.x"
-    MacOS Node 14:
-      Pool: Azure Pipelines
-      OSVmImage: "macOS-10.15"
-      TestType: "node"
-      NodeTestVersion: "14.x"
+
 
 jobs:
-  - job: "IntegrationTest"
+  - job:
```

Comment:
Is a name for the job not needed anymore?

Reply:
It ends up as a display name prefix that takes up all the space allotted in the UI, so that you have to mouse over to able to see the test parameters. It's kind of redundant anyway since we're not running other kinds of jobs in this pipeline.